package.json

```diff
@@ -43,6 +43,7 @@
     "broccoli-yuidoc": "^3.0.0",
     "chai": "^4.1.0",
     "cors": "^2.8.4",
+    "ember-ajax": "^3.0.0",
```

Comment:
This should not be needed

Reply:
![simple auth error](https://user-images.githubusercontent.com/471685/32164611-29a80d1a-bd47-11e7-9f42-3fcfbe1278fd.png)

Without it throw this error.

Reply:
Ah right, didn't realize we're using that in the demo 😞